tests/integration/models/reddit/test_redditor.py

```diff
@@ -123,20 +123,41 @@ def test_trophies__user_not_exist(self):
                 redditor.trophies()
             assert "USER_DOESNT_EXIST" == excinfo.value.error_type
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_trust__blocked_user(self, _):
+        self.reddit.read_only = False
+        with self.use_cassette():
+            redditor = self.reddit.user.blocked()[0]
+            with pytest.raises(RedditAPIException) as excinfo:
+                redditor.trust()
+            assert "CANT_WHITELIST_AN_ENEMY" == excinfo.value.error_type
+
     @mock.patch("time.sleep", return_value=None)
     def test_unblock(self, _):
         self.reddit.read_only = False
         with self.use_cassette():
             redditor = self.reddit.user.blocked()[0]
             redditor.unblock()
 
+    def test_trust(self):
+        self.reddit.read_only = False
+        with self.use_cassette():
+            self.reddit.redditor(self.FRIEND).trust()
+
```

Comment:
Kinda nitpicky as well, could you move this up to line 125 so it's sorted alphabetically?

Reply:
I chose this order so that all of the tests in this file could be re-run from scratch if needed. The first test, `test_block`, blocks PyAPITestUser3. That user remains blocked until the test `test_unblock` is run, which is why I placed  `test_trust__blocked_user` before `test_unblock`, and `test_trust` after it. 

I could change all three tests added tests so that they use a different user, block a user as part of the `test_trust__blocked_user` test, and rename `test_distrust` to `test_untrust`. Then the tests could be sorted alphabetically.

Reply:
Ah I see. The tests should be able recorded and ran in an arbitrary order. Take a look at this here: https://github.com/praw-dev/praw/blob/master/tests/integration/models/reddit/test_subreddit.py#L1650-1660

Reply:
would you like me to just re-arrange the tests so that they're sorted alphabetically? Or should I re-write `test_trust__blocked_user` so that a user is blocked before `trust()` is called?

Reply:
I would rearrange them and write the test to first block the user and then call `trust`. The tests should be able to be recorded on their own and pass.

Reply:
In that case, I'll do the PR to modify `block` first.